src/components/vuestic-components/va-radio/va-radio-docs.md

```diff
@@ -0,0 +1,20 @@
+# Radio Buttons
+
+```html
+<va-radio
+  v-model="selectedOptionString"
+  label="label"
+  name="radio"
+  color="success"
+  option="one"
+/>
+```
+
+### Props
+* `label`: String | Number - the label of Radio
+* `value`: amy - Model of the component; Either use this property (along with a listener for 'input' event) or use v-model directive
+* `option`: any - according to value for comparison, to determine whether the selected
```

Comment:
Hard to understand, maybe: "Option value that model is updated to when option is selected".